pkg/tuned/cmd/render/render.go

```diff
@@ -144,33 +169,55 @@ func render(inputDir []string, outputDir string, mcpName string) error {
 		return fmt.Errorf("Unable to get PerformanceProfile to apply using MachineConfigPool %s. error : %w", mcpName, err)
 	}
 
-	if len(filteredPerformanceProfiles) > 1 {
-		klog.Warningf("Found %d PerformanceProfiles for MachineConfigPool %s, only ONE is allowed.", len(filteredPerformanceProfiles), mcpName)
-		return fmt.Errorf("Found %d PerformanceProfiles for MachineConfigPool %s, only ONE is allowed.", len(filteredPerformanceProfiles), mcpName)
+	if len(filteredPerformanceProfiles) == 0 {
+		klog.Infof("No PerformanceProfile found for MachineConfigPool %s", mcpName)
 	}
-
-	if len(filteredPerformanceProfiles) > 0 {
-		perfProfile := filteredPerformanceProfiles[0]
-		tunedFromPP, err := tuned.NewNodePerformance(perfProfile)
+	for _, pp := range filteredPerformanceProfiles {
+		tunedFromPP, err := tuned.NewNodePerformance(pp)
 		if err != nil {
-			klog.Errorf("Unable to get tuned from PerformanceProfile %s. error : %v", perfProfile.Name, err)
-			return fmt.Errorf("unable to get tuned from PerformanceProfile:%s. error: %w", perfProfile.Name, err)
+			e := fmt.Errorf("unable to get tuned from PerformanceProfile:%s. error: %w", pp.Name, err)
+			klog.Error(e)
+			return e
 		}
 		// add tuned from PP to the list
 		tuneD = append(tuneD, tunedFromPP)
 	}
 
+	klog.Infof("working with %d additional tuneD profiles", len(tuneD))
+
 	tuneDrecommended := operator.TunedRecommend(tuneD)
 	if len(tuneDrecommended) == 0 {
-		klog.Error("Unable to get tuned recommended profile.")
-		return fmt.Errorf("Unable to get tuned recommended profile.")
+		klog.Infof("Unable to get tuned recommended profile with current info. Adding default tuneD")
+
+		fileName := "default-cr-tuned.yaml"
```

Comment:
You should always load this CR. Then you will not have to run the recommendation logic twice.

Reply:
thanks for the comment.
Done